README.md

```diff
@@ -1 +1,33 @@
 # bedrock-ledger-consensus-continuity-es-elector-pool
+
+## Install
+
+```
+npm i --save bedrock bedrock-ledger-node bedrock-ledger-utils
+npm i --save bedrock-ledger-consensus-continuity-es-elector-pool
+```
+
+## Usage
+
+In your ledger config:
+
+```js
+config['ledger-core'].config = { 
+  '@context': constants.WEB_LEDGER_CONTEXT_V1_URL,
+  type: 'WebLedgerConfiguration',
+  ledger: 'urn:ledger:36433d7c-a84d-4998-bb20-79e8581912ec',
+  consensusMethod: 'Continuity2017',
+  electorSelectionMethod: {
+    type: 'ElectorPoolElectorSelection',
+    electorPool: 'did:elector:060239ac-0631-4075-953a-7e61d9ee8c06'
```

Comment:
I don't know anything about `did:elector`, but in Veres One, a UUID DID like this is `did:v1:uuid:<uuid>`

https://github.com/veres-one/veres-one-validator/blob/master/schemas/did-uuid.js